test/integration/StrykerJestRunnerSpec.ts

```diff
@@ -6,19 +6,31 @@ import { expect } from 'chai';
 import JestTestRunner from '../../src/JestTestRunner';
 import * as path from 'path';
 
+// Get the project root, we will be stub process.cwd later on
+const jestProjectRoot = process.cwd();
+
+// Needed for Jest in order to run tests
+process.env.BABEL_ENV = 'test';
+
 describe('Integration StrykerJestRunner', function () {
```

Comment:
Can't you use an arrow function here?

Reply:
Without using `function` I cannot set the global timeout usint `this.timeout()`